addons/web/static/src/js/services/data_manager.js

```diff
@@ -137,10 +138,10 @@ return core.Class.extend({
      * @param {integer} params.actionId
      * @return {Promise} resolved with the requested filters
      */
-    load_filters: function (params) {
+    load_filters: async function (params) {
         var key = this._gen_key(params.modelName, params.actionId);
         if (config.isDebug('assets') || !this._cache.filters[key]) {
-            this._cache.filters[key] = rpc.query({
+            this._cache.filters[key] = await rpc.query({
```

Comment:
not sure about this... cache must contain promises to ensure not to do the same request twice